sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -513,10 +512,14 @@ kernel_pid_t gnrc_ipv6_netif_find_by_prefix(ipv6_addr_t **out, const ipv6_addr_t
         mutex_unlock(&(ipv6_ifs[i].mutex));
     }
 
+    if (out != NULL) {
+        *out = found;
```

Comment:
I see. It is called from here with `NULL`.
https://github.com/RIOT-OS/RIOT/blob/1ac9a0d227b452e117ed3f411933618a3dadf45f/sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c#L433
I thought accepting `NULL` as `out` is intended since the original code checks `out != NULL` before assignment. I will fix the call site, revert those hunks around `gnrc_ipv6_netif_find_by_prefix`, and remove `out != NULL` check in original code.